styles/layout.less

```diff
@@ -961,5 +961,22 @@ table.search-column {
     grid-row-gap: 2px;
 }
 
+/* ------------------------------------------------------------------------ */
+/* Project Smoothreading */
+
+.h2a {
```

Comment:
These class names aren't at all intuitive. Please rename them to something clearer.